apps/web/src/app/_styles/typography.css

```
@@ -152,6 +152,15 @@ html {
     }
   }
 
+  /* "#OpenWhatYouWant" hero */
+  .text-headline-back {
+    @apply text-sharp-33;
+    @variant landscape {
+      font-size: clamp(4rem, 8.5vw, 7.15rem); /* 114.4px */
+      line-height: 7.9375rem; /* 127px */
+    }
+  }
+
```

These values look off from design. Also, do not clamp.

```suggestion
  /* "#OpenWhatYouWant" hero */
  .text-headline-back {
    font-size: 1.8938125rem; /* 30.301px */
    line-height: 1.5673125rem; /* 25.077px */
    @variant landscape {
      font-size: 7.25rem; /* 116px */
      line-height: 6rem; /* 96px */
    }
  }

```

Also, these component classes are a perfect place to add text-specific styling like gradient text clipping if you want to keep it cleaner.